scripts/build.sh

```diff
@@ -11,7 +11,7 @@ else
 fi
 
 if [[ "$(git tag --points-at HEAD 2>/dev/null)" == v* ]]; then
-	touch "${ROOTDIR}/prerelease.txt"
+	rm "${ROOTDIR}/cmake/prerelease.lock"
```

Comment:
Shouldn't we rather use a cmake option instead of modifying the files?

Reply:
True. There's a `cmake` command below and we should just add the force release option there.